code/datums/antagonists/datum_cult.dm

```diff
@@ -5,13 +5,20 @@
 	qdel(communion)
 	return ..()
 
+/datum/antagonist/cult/can_be_owned(datum/mind/new_owner)
+	. = ..()
+	if(.)
+		. = is_convertable_to_cult(new_owner.current)
+
 /datum/antagonist/cult/on_gain()
 	. = ..()
-	if(!owner)
-		return
+	SSticker.mode.cult += owner
+	SSticker.mode.update_cult_icons_added(owner)
```

Comment:
it's almost like you removed every fucking line with SSticker in it! but that would be absurd and idiotic.